powershell/public/cisa/exchange/ConvertFrom-MailAuthenticationRecordDkim.ps1

```diff
@@ -40,7 +40,7 @@ function ConvertFrom-MailAuthenticationRecordDkim {
         [string]$DomainName,
 
         # DNS-server to use for lookup.
-        [ipaddress]$DnsServerIpAddress = "1.1.1.1",
+        [ipaddress]$DnsServerIpAddress,
```

Comment:
Is there any risk that `Resolve-DnsName` may return internal records for environments with a split-brain / split-horizon DNS configuration instead of relevant external DNS records?

Should we discuss the use of a default DNS provider? I do understand that some environments may also have external DNS resolution restricted to only allow certain machines to perform external recursion or may only allow it to certain public providers. How can we accommodate both scenarios and inform the user if something doesn't seem right?